tools/ci-cdk/canary-wasm/Cargo.toml

```diff
@@ -1,25 +0,0 @@
-
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
These deleted Cargo.* files should be added to the .gitignore so they don't get added back unintentionally.

Reply:
Added in bedff65